Core/src/org/sleuthkit/autopsy/datamodel/AbstractAbstractFileNode.java

```diff
@@ -127,194 +125,121 @@ private void updateSheet() {
+        }
+
+        @Override
+        public String toString() {
+            return displayString;
+        }
     }
 
     /**
      * Fill map with AbstractFile properties
      *
-     * @param map map with preserved ordering, where property names/values are
-     * put
-     * @param content to extract properties from
+     * @param map     map with preserved ordering, where property names/values
+     *                are put
      */
-    public static void fillPropertyMap(Map<String, Object> map, AbstractFile content) {
-
-        String path = "";
-        try {
-            path = content.getUniquePath();
-        } catch (TskCoreException ex) {
-            LOGGER.log(Level.SEVERE, "Except while calling Content.getUniquePath() on {0}", content); //NON-NLS
-        }
-
-        map.put(AbstractFilePropertyType.NAME.toString(), AbstractAbstractFileNode.getContentDisplayName(content));
-        map.put(AbstractFilePropertyType.LOCATION.toString(), path);
-        map.put(AbstractFilePropertyType.MOD_TIME.toString(), ContentUtils.getStringTime(content.getMtime(), content));
-        map.put(AbstractFilePropertyType.CHANGED_TIME.toString(), ContentUtils.getStringTime(content.getCtime(), content));
-        map.put(AbstractFilePropertyType.ACCESS_TIME.toString(), ContentUtils.getStringTime(content.getAtime(), content));
-        map.put(AbstractFilePropertyType.CREATED_TIME.toString(), ContentUtils.getStringTime(content.getCrtime(), content));
-        map.put(AbstractFilePropertyType.SIZE.toString(), content.getSize());
-        map.put(AbstractFilePropertyType.FLAGS_DIR.toString(), content.getDirFlagAsString());
-        map.put(AbstractFilePropertyType.FLAGS_META.toString(), content.getMetaFlagsAsString());
-        map.put(AbstractFilePropertyType.MODE.toString(), content.getModesAsString());
-        map.put(AbstractFilePropertyType.USER_ID.toString(), content.getUid());
-        map.put(AbstractFilePropertyType.GROUP_ID.toString(), content.getGid());
-        map.put(AbstractFilePropertyType.META_ADDR.toString(), content.getMetaAddr());
-        map.put(AbstractFilePropertyType.ATTR_ADDR.toString(), Long.toString(content.getAttrType().getValue()) + "-" + content.getAttributeId());
-        map.put(AbstractFilePropertyType.TYPE_DIR.toString(), content.getDirType().getLabel());
-        map.put(AbstractFilePropertyType.TYPE_META.toString(), content.getMetaType().toString());
-        map.put(AbstractFilePropertyType.KNOWN.toString(), content.getKnown().getName());
-        map.put(AbstractFilePropertyType.HASHSETS.toString(), getHashSetHitsForFile(content));
-        map.put(AbstractFilePropertyType.MD5HASH.toString(), content.getMd5Hash() == null ? "" : content.getMd5Hash());
-        map.put(AbstractFilePropertyType.ObjectID.toString(), content.getId());
-        map.put(AbstractFilePropertyType.MIMETYPE.toString(), content.getMIMEType() == null ? "" : content.getMIMEType());
+    void fillPropertyMap(Map<String, Object> map) {
```

Comment:
We are still statically accessing this method in KeywordSearchResultFactory with the old (Map, AbstractFile) arguments, which prevents Autopsy from building.